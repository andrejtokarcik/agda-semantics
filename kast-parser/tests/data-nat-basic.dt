data Nat : Set 0 where
  Zero : Nat,
  One : Nat,
  Succ : Pi x Nat Nat

; expr = (Lambda x Nat x) (Succ Zero)
