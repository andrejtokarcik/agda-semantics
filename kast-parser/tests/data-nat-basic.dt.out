<k>
    nf2 ( ( ((Lambda x Nat x)  Succ Zero ) , Nat ) )
</k>
<mgu>
    subst(.KList)
</mgu>
<tenv>
    .Map
</tenv>
<data>
    Nat |-> Set 0
    One |-> Nat
    Succ |-> (Pi x Nat Nat)
    Zero |-> Nat
</data>
<meta>
    .Map
</meta>
